Add Vulnerability Management WG (#151)
cloudfoundry/community: toc/working-groups/vulnerability-management.md

# Vulnerability Management: Working Group Charter

## Mission

Provides discreet management of security vulnerabilities issues relevant for active CF projects.

## Goals

> * Provides a single point of contact for security vulnerability reporting and management
> * Provides management of security vulnerability reports through to resolution, including but not limited to triage, reporter and team coordination, embargo negotiation, CVSS scoring, CVE assignments, pre-disclosure and disclosure

Author: ```suggestion
* Provide a single point of contact for security vulnerability reporting and management.
* Provide management of security vulnerability reports through to resolution, including but not limited to triage, reporter and team coordination, embargo negotiation, CVSS scoring, CVE assignments, pre-disclosure and disclosure.
```

## Scope

> * Triage incoming security vulnerability reports to [security@cloudfoundry.org](mailto:security@cloudfoundry.org)
> * Management of vulnerabilities through dedicated slack channels
> * When appropriate, negotiating suitable embargo periods with the reporter to afford component teams time to fix the issue before it becomes known publicly
> * When appropriate, assignment of CVE numbers to vulnerabilities/fixes
> * Publication of pre-disclosures to allow all CF editions time to adopt fixes for high/critical vulnerabilities before they become known publicly
> * Publication of disclosures

Author: ```suggestion
* Triage incoming security vulnerability reports to [security@cloudfoundry.org](mailto:security@cloudfoundry.org).
* Manage vulnerabilities through dedicated slack channels.
* When appropriate, negotiate suitable embargo periods with the reporter to afford component teams time to fix the issue before it becomes known publicly.
* When appropriate, assign CVE numbers to vulnerabilities/fixes.
* Publish pre-disclosures to allow all CF distributions time to adopt fixes for high/critical vulnerabilities before they become known publicly.
* Publish disclosures of reported security vulnerabilities.
```

## Non-Goals

> * Not adding security related features to Cloud Foundry

Author: ```suggestion
* Add security-related features to Cloud Foundry projects.
```

## Technical Lead(s): 

> - [Paul Warren](https://github.com/paulcwarren)

Reviewer: I think @paulcwarren is trying to get out of this role he's been playing. Do we have other ideas in mind for who could do this in the OSS community?

Reviewer 2: I don't know if he's found anybody as yet.

Reviewer 3: Sure.  Go ahead Lee.  For now, its me so merging is fine.

## Execution Lead(s):
- Monty Ijzerman
- Khushboo Soni

> - Matthias Essenpres 

Author: I think this is the correct spelling of this person's name, based on past emails, if I'm thinking of the right person.
```suggestion
- Mathias Essenpries 
```

> - Matthias Essenpres 

Author: ```suggestion
- Mathias Essenpreis 
```

Author: @stephanme or @loewenstein to confirm I have the spelling right this time 😅

> - Prateek Mehrotra

Author: We should add GitHub handles for these members.

> - Mathias Essenpreis 

Reviewer 2: Thank you Sir
